integration/loki_micro_services_test.go

```diff
@@ -126,4 +129,13 @@ func TestMicroServicesIngestQuery(t *testing.T) {
 		require.NoError(t, err)
 		assert.ElementsMatch(t, []string{"fake"}, resp)
 	})
+
+	t.Run("per-request-limits", func(t *testing.T) {
+		queryLimitsPolicy := client.InjectHeadersOption(map[string][]string{querylimits.HTTPHeaderQueryLimitsKey: {`{"maxQueryLength": "1m"}`}})
```

Comment:
am I missing some config that we have for this test client, I thought 1m was the default timeout, so this test should pass?

Reply:
It's the query length limit that defaults to sixty or thirty minutes. You see the change to the frontend config in line 69 above.

Reply:
I was missing that the setup of the `cliQueryFrontendLimited` and calling `cliQueryFrontendLimited.LabelNames` uses a time of `30 * time.Minute` defined in `integration/client.go` :+1: